config/routes.rb

```diff
@@ -38,6 +38,13 @@
     end
   end
 
+  resources :collections do
+    member do
+      get :manifest, defaults: { format: :json }, as: "manifest_curation_concerns"
+      get :manifest, defaults: { format: :json }
```

Comment:
It's actually on purpose, because the helpers resolve to two things in different places, since I could never get polymorphic_url to work in the builder. I could fix it by passing in the manifest url, maybe?